mozregression/cli.py

```diff
@@ -265,6 +265,12 @@ def create_parser(defaults):
                              " release using a specific release number, and"
                              " a nightly using a specific buildid")
 
+    parser.add_argument('-P', '--process-output', choices=('0', '1'),
+                        default=defaults['process-output'],
+                        help=("Manage process output logging. 1 to enable, 0"
```

Comment:
Basically to allow the user to enable this option in the config file, but still allowing him to disable it from the command line.

store_true is not something that can be easily overridden from the command line when you have it defined in the config file. it would require a -\-no-output to deactivate it, and I don't see any simple way to manage it with argparse. I tend to use now argument that always take a parameter, so we can override the value easily when it is defined in the config file.
Make sense ? If you have another idea, tell me. :)